moped-editor/src/views/projects/projectsListView/ProjectsListViewQueryConf.js

```diff
@@ -52,15 +55,27 @@ export const ProjectsListViewQueryConf = {
     project_name: {
       searchable: true,
       sortable: false,
+      link: "project_id",
```

Comment:
the name of the column that has the link url information, in this case the project_id pk